client/verta/verta/_internal_utils/access_token.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+
+class AccessToken:
```

Comment:
Do you think this might be generalizable to other parts of our platform? Otherwise I'd be temped to define this class in `verta.endpoint` (or `verta.deployment`, though I'd like to merge it into `verta.endpoint` soon™).

Reply:
I actually put it in `verta.endpoint` originally, but some import circularity forced me to move it out of there. If we could consolidate `DeployedModel` into `verta.endpoint` that might get around the issue.